### PR TITLE
[WIP] Update Travis build

### PR DESCRIPTION
**Pull request by jopasserat**
Generate static builds upon success using the new [build environment container](https://github.com/proot-me/proot-static-build/tree/build_environment)

### CODE REVIEW

**Comment by vincenthage** [comment]
It looks like Travis doesn't run tests on some PRs, like [this one](https://travis-ci.org/proot-me/PRoot/builds/272982213?utm_source=github_status&utm_medium=notification).
There might have a fix meanwhile that fixed them.

**Comment by jopasserat** [comment]
Yes you're right there was a mistake on the command line for the FHS build, the tests didn't run:
https://travis-ci.org/proot-me/PRoot/builds/272982213#L564

**Comment by vincenthage** [comment]
I've set the value of `COVERITY_SCAN_BRANCH` to 0 globally on Travis' settings.
It should take care of builds for which the coverage can't be done.

So you should re-enable in this PR so that it's at least run on the master branch (it should override the global value).

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
 
 before_install:
  - sudo apt-get update -qq
- - sudo apt-get install -qq libtalloc-dev uthash-dev libarchive-dev gdb strace realpath
- - sudo pip install cpp-coveralls
+ - sudo apt-get install -qq libtalloc-dev libarchive-dev uthash-dev gdb strace realpath
+ - sudo -H pip install cpp-coveralls==0.3.12
 
-script: if [ ${COVERITY_SCAN_BRANCH} != 1 ]; then make -C src loader.exe loader-m32.exe build.h && env CFLAGS=--coverage LDFLAGS='--coverage' make -C src proot && env PATH=/bin:/usr/bin:/sbin:/usr/sbin:$PWD/src make -C tests; fi
+script: make -C src loader.exe loader-m32.exe build.h && env CFLAGS=--coverage LDFLAGS='--coverage' make -C src proot care && env PATH=/bin:/usr/bin:/sbin:/usr/sbin:$PWD/src PROOT_NO_SECCOMP=1 make -C tests
```
Comment (vincenthage) [inline .travis.yml]: The Seccomp mode should be enabled for the tests.

Comment (jopasserat) [reply .travis.yml]: yes debug until we merge #130 ;)

**Pull request merged**